src/Components/Home.js

```diff
@@ -2,21 +2,37 @@ import './Home.css';
 import { useState } from 'react';
 
 const Home = () => {
-    const [value, setValue] = useState('');
+    const initialValue = [];
+    const [value, setValue] = useState([]);
+    const [initial, setInitial] = useState(false);
     var val = '';
     const onChange = (e) => {
         val = e.target.value;
-        console.log(val);
+        if (initial == true) {
+            e.target.value = "";
```

Comment:
clear this state, as soon as the item is added to the list. Currently, when a user attempts to add another item, it resets the state.